src/chrome/content/quotecollapse/quotecollapse.js

```diff
@@ -72,16 +72,29 @@ blockquote[type="cite"][qctoggled="true"] {\n\
  max-height: none;\n\
  overflow: visible;\n\
 }\n\
-\n\
-blockquote[type="cite"]:not([qctoggled="true"]) blockquote[type="cite"] {\n\
- background-image: url("chrome://quotecollapse/skin/twisty-clsd.png");\n\
- max-height: 2.25ex;\n\
- overflow: -moz-hidden-unscrollable;\n\
-}\n\
 ';
     var styletext = document.createTextNode(stylecontent);
     StyleElement.appendChild(styletext);
     messageDocument.getElementsByTagName("head").item(0).appendChild(StyleElement);
+
+    for(let quote of messageDocument.querySelectorAll("blockquote")) {
+      QuoteCollapse._toggleFullyVisible(quote);
+    }
+  },
+  
+  _toggleFullyVisible: function toggleFullyVisible(quote) {
+    if(quote.getAttribute("qctoggled") == "true")
+      return true;
```

Comment:
I'll remove that `if` bit – causes somewhat unexpected behavior in certain situations for me.  It wasn't in the original patch, also.